iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/execution/QueryExecution.java

```diff
@@ -353,10 +355,14 @@ private void schedule() {
   // Use LogicalPlanner to do the logical query plan and logical optimization
   public void doLogicalPlan() {
     LogicalPlanner planner = new LogicalPlanner(this.context);
+    long startTime = System.nanoTime();
     this.logicalPlan = planner.plan(this.analysis);
```

Comment:
Now there will be a new timer called optimize time in LogicalPlanner. Divide these two.